### PR TITLE
Staking video component

### PR DESCRIPTION
**Pull request by kimpers**
This re-uses the ModalVideo component for the staking hero header.

image

### CODE REVIEW

File: ts/pages/staking/home.tsx [hunk 1]
```diff
@@ -17,6 +17,8 @@ export const StakingIndex: React.FC<StakingIndexProps> = props => {
                 titleMobile="Start staking your tokens"
                 description="Use one pool of capital across multiple relayers to trade against a large group"
                 figure={<CFLMetrics />}
+                // TODO(kimpers): Add correct video for staking portal
+                videoId="c04eIt3FQ5I"
```
Comment (kimpers) [inline ts/pages/staking/home.tsx:21]: Need to update this with the real video once I have that. Used the homepage video as a placeholder

**Comment by chriskalani** [comment]
👍

**Comment by kimpers** [comment]
@chriskalani do you know if there is a video prepared already or should I leave it with the placeholder video for now?

**Comment by chriskalani** [comment]
A placeholder is fine. We haven't started production on the video yet.

**Pull request merged**